android/src/main/java/io/mosip/tuvali/openid4vpble/Openid4vpBleModule.kt

```diff
@@ -62,7 +62,7 @@ class Openid4vpBleModule(private val reactContext: ReactApplicationContext) :
     }.orEmpty()
   }
 
-  private fun onException(exception: Throwable){
+  private fun onException(exception: Exception){
     if(exception.cause != null){
```

Comment:
I know this is not changed today, but is this logic really required? Can we remove it if not required?